cloudevents-sdk-actix-web/src/server_request.rs

```diff
@@ -111,6 +112,25 @@ pub async fn request_to_event(
         .map_err(actix_web::error::ErrorBadRequest)
 }
 
+/// Extention Trait for [`HttpRequest`]which acts as a wrapper for the function [`request_to_event()`]
+#[async_trait(?Send)]
```

Comment:
Can you check both here and in `HttpResponseBuilderExt` if it breaks to remove the `?Send`? Let's add it only if we need it.

Reply:
Yeah it brakes unfortunately without ```?Send```.